### DISCUSSION TITLE
Private Repository Contributions Not Showing in GitHub Profile

### DISCUSSION THREAD

**Discussion by harish18092002** [community/community#138925] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I've noticed that my contributions to the main branch of a private repository are not appearing in my GitHub contribution history. I've already checked my email settings and other common troubleshooting steps. Looking for help to understand why this is happening and how to resolve it.

## My Commit history

<img width="913" alt="Screenshot 2024-09-17 at 11 30 41 PM" src="https://github.com/user-attachments/assets/76c7692d-7713-408b-a3c0-1bdede93120b">

## My contributions history

<img width="1507" alt="Screenshot 2024-09-17 at 11 31 41 PM" src="https://github.com/user-attachments/assets/8c0246ef-639e-4c06-82f5-ff7bc7a47171">

**Comment by davevad93** [comment]
Hi @harish18092002 ,

The problem seems to be that the commits aren't recognized as yours, as you can see in your screenshot. your last 2 commits are linked to a non-existent account "Harish" with a blank image, that usually happens when there's a mismatch with the `user.email` or the `SSH` key used to push the changes.

Since your prior commit appears correctly I wonder, have you made any changes since yesterday, e.g. pushed your changes from another computer or used a new `SSH` key?

**Reply by harish18092002** [reply]
I appreciate your response. It was beneficial, and the issue has been resolved. Thank you for your assistance.

**Reply by davevad93** [reply]
You're welcome @harish18092002 , glad you fixed it! 👍
